### ISSUE TITLE
[Share with users, groups] Should not be accent nor case sensitive

### ISSUE THREAD

**Issue by etiess** [owncloud/client#4284]
The new option in the context menu to share a file/folder with users/groups is great. I just noticed that it is case sensitive and acccent sensitive. I think it should not.

Step to reproduce to share a file with a user called Frédéric :
1. Right click on a file in your file explorer
2. Share with ownCloud
3. Start entering the name of a user: "fr..."
4. The name Frédéric should be proposed, but it's not.

In fact, you have to enter the name with a capital letter, in order it is proposed in the users list. 

It is the same if I start entering "Fred..." (instead of "Fréd..."). 

And it's the same if I use his "login" instead of his name: for example, if Frédéric's login is "fredo", I should be able to enter "fredo" in the box.

To summarize, entering a name in the sharing box should:
1. Not be sensitive to capital letters nor accentuated letters 
2. Consider both the name and the login of a user

Version used: 2.1.0 (build 5683) on Windows 10
Server: 8.1.2 on Ubuntu 15.04 (MySQL)

**Comment by RealRancor** [comment]
@etiess That should be a dup of https://github.com/owncloud/client/issues/4269 which gets fixed with 2.1.1

**Comment by stonerl** [comment]
@etiedd please try the latest nightly. The case sensivity bug is fixed there. But I don't know about accents.

**Comment by etiess** [comment]
@RealRancor : Thank you, I didn't look into closed issues, sorry for the dup.
@stonerl : unfortunately, I can't test the latest nightly, I just have one computer with owncloud installed and in production. So I can't test if accents are solved. I modify this issue to specifically target accents. And by the way, I'll create an issue on the server side, as sharing on the web interface is accent sensitive, while it shouldn't (in my opinion).

**Comment by etiess** [comment]
Closed as a partial dup of https://github.com/owncloud/client/issues/4269

New client issue here: https://github.com/owncloud/client/issues/4286
New server issue here: https://github.com/owncloud/core/issues/21212
